### ISSUE TITLE
Accessibility Feedback - Midpoint Review - [DGIB, Enrollment Verification]

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#33397]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Even if your project requires minimal verification of functionality, it is prudent to develop a test plan that ensures quality is baked into your product. You have user stories which are a great way to call your shots. Now, you need to develop a test plan that verifies your user stories have been developed to satisfy functional requirements and business needs.

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
List of enrollment verifications with screen reader should read like a list, with total amount and current one selected

**Should:**

**Consider:**

**Comment by rlyons89** [comment]
@noahgelman, just want to make sure that this feedback is directing us to make sure a <ul> tag structure is used for the html for the enrollment verifications list?

**Comment by noahgelman** [comment]
@rlyons89 Yes, or whatever html structure is used in similar looking content in other places on VA. I just to make sure it's read correctly by screen readers.
